apps/encryption/lib/Command/FixEncryptedVersion.php

```diff
@@ -185,6 +175,22 @@ private function verifyFileContent($path, OutputInterface $output, $ignoreCorrec
 			$handle = $this->view->fopen($path, 'rb');
 
 			if (\fread($handle, 9001) !== false) {
+				$fileInfo = $this->view->getFileInfo($path);
+				if (!$fileInfo) {
+					$output->writeln("<warning>File info not found for file: \"$path\"</warning>");
```

Comment:
Seems more appropriate:

```suggestion
					$output->writeln("<warning>Unknown file: \"$path\"</warning>");
```